styles.css

```diff
@@ -65,3 +71,147 @@ ul {
 li {
   margin: 5px;
 }
+
+#work-item-one {
+  height: 540px;
+  padding: 20px;
+  padding-right: 0;
+  background-color: rgb(255, 255, 255);
+  border-radius: 10px;
+  border: 1px solid rgb(220, 217, 217);
+  padding-bottom: 40px;
+}
+
+#work-item-one h1 {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  margin: 1px 0 2px;
+  width: 186px;
+}
+
+#work-item-one p {
+  height: 62px;
+  width: 327px;
+  left: 24px;
+  top: 276px;
+  font-family: "Poppins", sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  letter-spacing: 0;
+  text-align: left;
+}
+
+#work-item-one span {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  border: 1px solid #396df2;
+  border-radius: 16px;
+  width: 140px;
+  height: 48px;
+  color: #396df2;
+}
+
+#work-item-one ul {
+  display: flex;
+  flex-direction: row;
+  padding-right: 20px;
+  font-family: "poppins", sans-serif;
+  font-weight: 500;
+}
+
+#canopy {
+  font-weight: 600;
+  color: #404752;
+}
+
+#first-list {
+  display: flex;
+  color: #4c5058;
+  padding-left: 0;
+  margin: 0;
+}
+
+#first-list img {
+  padding-right: 10px;
+}
+
+#languages li {
+  background-color: #ebebff;
+  padding: 0 10px;
+  border-radius: 5px;
+  width: max-content;
+  height: 30px;
+  justify-content: center;
+  text-align: center;
+  color: #6070ff;
+  margin-top: 1px;
+}
+
+#About-Myself span {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  border: 1px solid #396df2;
+  border-radius: 16px;
+  width: 140px;
+  height: 48px;
+  color: #396df2;
+}
+
+#Languages span {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  flex-direction: row;
+  border-bottom: 1px solid rgb(187, 187, 187);
+  width: 359px;
+  font-family: "Poppins", sans-serif;
+}
```

Comment:
- [ ] [REQUIRED] I appreciate your hard work 👍, but there is a bit of confusion in the placement of the `Frameworks` and `Skills` part. Please note that the `text`(`Frameworks`, `Skills`) part of this section should be aligned on the `left` side, and the same problem with the `arrow` icons, they should also be aligned on the right side like a [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)😃 

![image](https://user-images.githubusercontent.com/63915024/198705222-82faa053-c67c-468f-9116-9cbefb1e833e.png)